extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -305,6 +363,135 @@ describes.realWin(
         await macroTask();
         expect(await ampConsent.getConsentRequiredPromise_()).to.be.true;
       });
+
+      describe('remote server response', () => {
+        it('does not update local storage when response is false', async () => {
+          const remoteConfig = {
+            'consentInstanceId': 'abc',
+            'consentRequired': 'remote',
+            'checkConsentHref': 'https://server-test-1/',
```

Comment:
for readability, move the fake response object into this test method as a local var.